src/main/java/com/AdelaQA/classesandobjects/autoservicetask/Tool.java

```diff
@@ -0,0 +1,18 @@
+package com.AdelaQA.classesandobjects.autoservicetask;
+
+public class Tool {
+    double price;
+    String name;
+boolean isUsed;
+double weight;
+
+public Tool(String name){
+    this.name = name;
+    System.out.println("Un obiect de tip tool a fost creat, numele lui este " + this.name);
+}
+public Tool(){
```

Comment:
you are using parameters that you didn't initialize in the constructor
public Tool(String name, double weight){
}